queenbee/io/common.py

```diff
@@ -0,0 +1,133 @@
+"""common objects between different IO files."""
+from enum import Enum
+from typing import Dict, List, Any
+
+from pydantic import constr, Field, validator
+from jsonschema import validate as json_schema_validator
+
+from .reference import FolderReference, FileReference, references_from_string
+from ..base.basemodel import BaseModel
+
+
+class ItemType(str, Enum):
+    """Type enum for items in a list."""
+    String = 'string'
+    Integer = 'integer'
+    Number = 'number'
+    Boolean = 'boolean'
+    Folder = 'folder'
+    Array = 'array'
+    Object = 'object'
```

Comment:
These reserved keywords works fine with Enum in C# with CamelCase.
![image](https://user-images.githubusercontent.com/9031066/94933786-dc26c300-0498-11eb-87c1-6c81de19a730.png)

But not sure if this order works:
String = 'string'  (CamelCase = 'lowercase')
 
In [Honeybee schema, it is flipped](https://github.com/ladybug-tools/honeybee-schema/blob/071e5e3d5200a435f9970e4236ff6c892d96e0dd/honeybee_schema/energy/designday.py#L156-L167):
![image](https://user-images.githubusercontent.com/9031066/94934194-65d69080-0499-11eb-83aa-a9a1c8776bf0.png)

Reply:
Thanks. Will make the change.

Reply:
The change has been made.